packages/definitions-parser/src/lib/definition-parser.ts

```diff
@@ -154,6 +154,32 @@ export function parseVersionFromDirectoryName(directoryName: string | undefined)
   };
 }
 
+/**
+ * Like `parseVersionFromDirectoryName`, but the leading 'v' is optional,
```

Comment:
The root cause of a lot of version numbers getting lost was plugging a string like `"17"` into this function, where I was expecting to get back `{ major: 17, minor: undefined }`, but instead got back `undefined` because this function expects the format `v17`, since it was written only for directory names.